.github/workflows/nightly.yml

```diff
@@ -0,0 +1,56 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  tests_release:
+    name: Test Release
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      # Linux
+      linux_env_vars: |
+        NODE_VERSION=v18.19.0
+        NODE_PATH=/usr/local/nvm/versions/node/v18.19.0/bin
+        NVM_DIR=/usr/local/nvm
+        CI=1
+      linux_pre_build_command: |
+        apt-get update && apt-get install -y rsync curl gpg libasound2 libgbm1 libgtk-3-0 libnss3 xvfb build-essential
+        mkdir -p $NVM_DIR
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+        /bin/bash -c "source $NVM_DIR/nvm.sh && nvm install $NODE_VERSION"
+        echo "$NODE_PATH" >> $GITHUB_PATH
```

Comment:
Should this be updated to 
```
linux_pre_build_command: . .github/workflows/scripts/setup-linux.sh
```

Reply:
yep, thanks! Intent was to share but forgot to update